xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -500,7 +500,7 @@ The array must already be pinned before this method is called, and that array mu
       </Docs>
     </Member>
     <Member MemberName="CreateReadOnlySpan&lt;T&gt;">
-      <MemberSignature Language="C#" Value="public static ReadOnlySpan&lt;T&gt; CreateReadOnlySpan&lt;T&gt; (scoped ref T reference, int length);" />
+      <MemberSignature Language="C#" Value="public static ReadOnlySpan&lt;T&gt; CreateReadOnlySpan&lt;T&gt; (ref T reference, int length);" />
```

Comment:
```suggestion
      <MemberSignature Language="C#" Value="public static ReadOnlySpan&lt;T&gt; CreateReadOnlySpan&lt;T&gt; (scoped ref T reference, int length);" />
```

Reply:
This change should be reverted.

Reply:
I wonder why mdoc added that. The source code in the preview6 branch does not include the `scoped` keyword:

https://github.com/dotnet/runtime/blob/d3fa592f6d5d678edb8aea7568812b167c72e622/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.cs#L234

Reply:
mdoc didn't add it - Aaron did :) See https://github.com/dotnet/dotnet-api-docs/pull/8187#issuecomment-1172452681.

Reply:
My memory sucks. I even reviewed it. 😄